common/environment/configure/gnu-configure-args.sh

```diff
@@ -82,6 +82,12 @@ case "$XBPS_TARGET_MACHINE" in
 		. ${_AUTOCONFCACHEDIR}/mips-linux
 		;;
 
+
+	ppc|ppc-musl)
+		. ${_AUTOCONFCACHEDIR}/endian-big
```

Comment:
should problably generate ppc-common here if applicable

Reply:
@maxice8 Sorry, but I don’t really understand what you mean with “generate ppc-common”. Could you please elaborate a bit? :)